apps/src/levelbuilder/ai-tutor/constants.ts

```diff
@@ -32,6 +32,10 @@ const genAIEndpoints = [
     id: AiChatModelIds.PIRATE,
     name: 'Mistral Pirate + Webpurify',
   },
+  {
```

Comment:
Not adding this made TS mad, and @Erin007 informed me that the usage of this constant is going away in this PR, so adding it in the short term (only used by an internal tool):

https://github.com/code-dot-org/code-dot-org/pull/63368